components/script_bindings/codegen/run.py

```diff
@@ -153,9 +153,6 @@ def attribute_names(property_name: str) -> Iterator[str]:
     if "-" in property_name:
         yield "".join(camel_case(property_name))
 
-    # https://drafts.csswg.org/cssom/#dom-cssstyledeclaration-webkit-cased-attribute
-    if property_name.startswith("-webkit-"):
-        yield "".join(camel_case(property_name, True))
```

Comment:
Can you remove the `webkit_prefixed` argument from `camel_case` since it's unused now?